### PR TITLE
Improve the email report and minor fixes

### PR DESCRIPTION
**Pull request by rbalint**
The kept packages would look like that in the email:
...
Packages with upgradable origin but kept back:
 Ubuntu cosmic-security:
  libavahi-client3
 LP-PPA-rbalint-scratch:
  other-package
...

### CODE REVIEW

File: unattended-upgrade [hunk 1]
```diff
@@ -453,6 +454,38 @@ class Unlocked:
...
+    """
+    def add(self, pkg, cache):
+        # type: (apt.Package, UnattendedUpgradesCache) -> None
+        allowed_origins = cache.allowed_origins
+        for origin in ver_in_allowed_origin(pkg, allowed_origins).origins:
+            if is_allowed_origin(origin, allowed_origins):
+                self[origin.origin + " " + origin.archive].add(pkg.name)
+                return
+
+    def pop_upgradable(self, cache):
```
Comment (julian-klode) [inline unattended-upgrade]: This is confusing, why would I want to pop packages that are upgradable from a dict origin -> package? Maybe the class needs a better name that explains the purpose better

File: unattended-upgrade [hunk 2]
```diff
@@ -453,6 +456,38 @@ class Unlocked:
...
```
Comment (julian-klode) [inline unattended-upgrade]: This is confusing, why is that needed? 

Answer: Because we only want to list kept-back packages; but this is not explicit in the name of the class (or method?)

Comment (rbalint) [reply unattended-upgrade]: How about naming the class KeptPackages and adding some comments? Do you have a better idea for the name?

Comment (rbalint) [reply unattended-upgrade]: @julian-klode I tried to clear the confusion, do you think it is good now?

Comment (julian-klode) [reply unattended-upgrade]: Yeah
